Drop hardcoded dependency on a wallpaper

joukewitteveen opened the pull request:
This sets the default to a solid blue color. A custom wallpaper can be used by
starting weston via
`BACKGROUND=/usr/share/wallpapers/Hanami/contents/images/3872x2592.jpg weston`

This fixes #32.

MrSchism commented:
It seems to work for me.

fooishbar commented:
Thanks for this, and sorry for the slow response. Would it be possible to implement this as a fallback, such that the default is retained, but when not available it falls back to a solid colour, rather than exiting?

joukewitteveen commented:
I updated the PR so that it defaults to the original wallpaper when started through `maynard`. This keeps the code nice and tidy: the binary is still free of a hardcoded path.

joukewitteveen commented:
I rebased the commits onto the current master and added support for specifying MAYNARD_BACKGROUND="" (empty string), which suppresses the default background.
